spec/controllers/tags_controller_spec.rb

```diff
@@ -93,22 +93,39 @@ def create_tags
         expect(assigns(:galleries)).to match_array([gallery])
       end
 
-      it "succeeds with valid character tag" do
+      it "gallery group succeeds with valid character tag" do
```

Comment:
To be more correct, I think this "gallery group" bit should be pulled out into a separate context. (Currently it's not grammatical – "it gallery group succeeds with valid character tag", whereas "context gallery group, it succeeds with valid character tag" is more so.)